lib/postgrex/connection.ex

```diff
@@ -85,16 +85,38 @@ defmodule Postgrex.Connection do
     message = {:query, statement, params}
     timeout = opts[:timeout] || @timeout
     case Connection.call(pid, message, timeout) do
-      %Postgrex.Result{} = res ->
-        {:ok, res}
-      %Postgrex.Error{} = err ->
-        {:error, err}
       {:error, kind, reason, stack} ->
         :erlang.raise(kind, reason, stack)
```

Comment:
Raising an exception with a stack trace from another process? Please let's not do this as the stack trace is lying and going to confuse. Not related to this issue though.